app/build.gradle

```diff
@@ -112,7 +113,8 @@ android {
 
     buildTypes {
         release {
-            minifyEnabled false // See https://stackoverflow.com/questions/40232404/google-play-apk-and-android-studio-apk-usb-debug-behaving-differently - proguard.cfg modification alone insufficient.
+            minifyEnabled false
```

Comment:
As I see such formatting changes at multiple places so it would be a good idea to review that your code formatting settings are set correctly. Ideally, you would want to auto format just the uncommitted changes ie. option 1, not the whole file. 

<img width="339" alt="screen shot 2018-05-18 at 1 20 34 am" src="https://user-images.githubusercontent.com/3069373/40200093-15106728-5a39-11e8-855c-287fed73217b.png">

Am stressing this point as otherwise merge conflicts would become a nightmare with multiple people changing these files. :)

Reply:
Yes I will be quite helpful. I will immediately update my auto format settings. Thanks a ton !